### PR TITLE
Allow users to turn on access to cluster via GCP public IP address space

### PR DESCRIPTION
**Pull request by ankitkinra**
Allows user to enable access to GKE cluster via GCP public IP address space

* Added a new variable (false by default) in the GKE cluster module
* After this change user can access the GKE cluster (if this variable is turned on) if they are trying to access cluster from their GCP hosted workstation of GCP VM.

### CODE REVIEW

File: community/examples/ml-gke.yaml
Comment (nick-stroud) [inline community/examples/ml-gke.yaml:1]: Please add description to PR. 

- Explain or link to documentation on what access this feature allows. It sounds like I could have another VM in google cloud and then I could access GKE API from that VM.
- What manual testing have you done on this change?

File: community/examples/ml-gke.yaml [hunk 1]
```diff
@@ -22,10 +22,13 @@ vars:
   zones:
   - asia-southeast1-b  # g2 machine has better availability in this zone
 
-  # Cidr block containing the IP of the machine calling terraform.
-  # The following line must be updated for this example to work.
+  # Uncomment the following lines if you want to add
+  # cidr block containing the IP of the machine calling terraform.
   authorized_cidr: <your-ip-address>/32
```
Comment (nick-stroud) [inline community/examples/ml-gke.yaml:27]: new comment above says "uncomment" but the authorized cidr is not commented out.

**Review by nick-stroud — CHANGES_REQUESTED** [review]

File: community/examples/ml-gke.yaml
Comment (ankitkinra) [reply community/examples/ml-gke.yaml:1]: Update the PR description.

For manual testing I added the variable in the blueprint, deployed the cluster and tested that my GCP cloud workstation was able to access the cluster without explicitly adding my workstation ip address

File: community/examples/ml-gke.yaml (hunk 1 above)
Comment (ankitkinra) [reply community/examples/ml-gke.yaml]: Updated and removed the changes to blueprint.

File: community/examples/ml-gke.yaml
Comment (nick-stroud) [reply community/examples/ml-gke.yaml:1]: PS. My comment here was to actually edit the PR description to add this information.

**Review by nick-stroud — APPROVED** [review]

**Pull request merged**